doc/api/child_process.md

```diff
@@ -147,7 +147,9 @@ added: v0.1.90
 * Returns: {ChildProcess}
 
 Spawns a shell then executes the `command` within that shell, buffering any
-generated output.
+generated output. When specifying paths, spaces need to be dealt with 
+accordingly, eg. `exec("'/path/to/test file/test.sh' arg1 arg2")` or 
```

Comment:
Our code standards use single quotes instead of double quotes and I think all of our examples follow that style too (except perhaps in cases where double quote avoids escaping). 

I would recommend avoiding `e.g.` and just spelling out `for example`. Even better perhaps, just remove it entirely and put a colon after `accordingly`. Then put the examples on separate lines by themselves. 

> ...dealt with accordingly:
>
>   ```
>    exec('"path...arg2');
>    exec('"path/to/test...arg2');
>  ```

Ping @nodejs/documentation: Is using a code block that way OK when providing examples like that? I mean, it's not *really* a single block of code, but I'm not sure how else to do it? (Each one in its own code block?_